pkg/document/tree_content_identity_test.go

```
@@ -0,0 +1,188 @@
+
+	doc := document.New("doc")
+	actor, err := time.ActorIDFromHex(hexActor)
+	assert.NoError(t, err)
+	doc.SetActor(actor)
+
+	assert.NoError(t, doc.Update(func(r *json.Object, p *presence.Presence) error {
+		r.SetNewTree("t", json.TreeNode{
+			Type: "r",
+			Children: []json.TreeNode{{
+				Type:     "p",
+				Children: []json.TreeNode{{Type: "text", Value: "ab"}},
```

golangci-lint (2.12.2)

[error] string `text` has 15 occurrences, make it a constant

(goconst)